core/src/test/java/com/novoda/noplayer/mediaplayer/ExceptionMatcher.java

```diff
@@ -0,0 +1,28 @@
+package com.novoda.noplayer.mediaplayer;
+
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+
+class ExceptionMatcher extends BaseMatcher<Exception> {
```

Comment:
I thought it was implied by the static factory method, I am happy to change it if you think it could be more explicit.